docs/ios-sdk.md

```diff
@@ -434,31 +389,10 @@ The player will receive the millisecond precision server time from the time urls
 
 In other words, IOS SDK players behave always in sync once initialized, meaning if audio packages or video frames are received later then their presentation timestamps due the network connections, they will be ignored or played back in a faster rate by the player.
 
-## ABR (Adaptive BitRate)
-
-Current version players just select the highest quality video for ABR
-
 ## Error Handling
 
 To be updated with recovery options implementation in v 1.0.0
```

Comment:
Is this still something we need to do or can improve?
If so, could you create a ticket for it?

Reply:
No, error handling is implemented. We can remove it from the docs.

Reply:
ok